_data/banking.yml

```diff
@@ -374,6 +374,7 @@ websites:
       img: natwest.png
       tfa: Yes
       hardware: Yes
+      software: Yes
       doc: http://www.natwest.com/personal/online-banking/g1/banking-safely-online/card-reader.ashx
```

Comment:
we try to do limit site addition PRs to not edit existing sites, please remove changes